radicle-cli/examples/workflow/5-patching-maintainer.md

```diff
@@ -61,3 +61,18 @@ a07ef7743a3 R1 (27857ec) -> R2 (f6484e0)
 ✓ Patch a07ef7743a32a2e902672ea3526d1db6ee08108a updated 🌱
 
 ```
+
+```
+$ git checkout master
+Your branch is up to date with 'rad/master'.
+$ rad merge a07ef7743a32a2e902672ea3526d1db6ee08108a
+Merging a07ef7743a3 Ra07ef7743a32a2e902672ea3526d1db6ee08108a (3e674d1) by did:key:z6Mkt67GdsW7715MEfRuP4pSZxJRJh6kj6Y48WRqVv4N1tRk into master (f2de534) via fast-forward...
+Running `git merge --ff-only 3e674d1a1df90807e934f9ae5da2591dd6848a33`...
+Updating f2de534..3e674d1
+Fast-forward
+ REQUIREMENTS | 0
+ 1 file changed, 0 insertions(+), 0 deletions(-)
+ create mode 100644 REQUIREMENTS
+✓ Updated master f2de534 -> 3e674d1 via fast-forward
+✓ Patch state updated, use `rad push` to publish
+```
```

Comment:
We should test that the merge actually happened :)

Reply:
You could do:
```
$ git show-ref refs/heads/master
f6484e0f43e48a8983b9b39bf9bd4cd889f1d520 refs/heads/master
```